apps/mail-bridge/env.ts

```diff
@@ -69,7 +69,9 @@ export const env = createEnv({
       })
     ),
     MAILBRIDGE_POSTAL_SERVERS_DNS_ROOT_URL: z.string().min(1),
-    MAILBRIDGE_POSTAL_LOCAL_MODE: z.coerce.boolean().default(false),
```

Comment:
why was this changed?
it should be local mode based on other env vars, not based on billing module

Reply:
`z.coerce` doesn't behave as I expected. So I thought to make it based on EE license. I found a way to handle it properly now. So I have updated it.